### ISSUE TITLE
move ember dependency into optionalDependencies

### ISSUE THREAD

**Issue by CNDW** [components/ember-data#35]
having ember in the dependencies forces a npm install of the components repo to have a duplicate install of ember. If someone is installing ember/ember-data via the components shim library, their package.json will have to look something like this

```
    "components-ember": "https://github.com/components/ember/archive/2.4.2.tar.gz",
    "components-ember-data": "https://github.com/components/ember-data/archive/2.4.0.tar.gz",
    "ember": "https://github.com/emberjs/ember.js/archive/v2.4.2.tar.gz"
```

The ember library is currently incompatible with npm, which is the whole reason for using the shims.

An alternative would be to have the 'components-ember' repo in the dependencies key.

**Comment by bmac** [comment]
Seems like a good change. @CNDW would you have time to make a pull request to update this file https://github.com/emberjs/data/blob/master/config/package-manager-files/package.json in the ember data repo which is used to generate the package.json in this repo?

**Comment by CNDW** [comment]
I can do that, any opinion on which route to take? optionalDependency or `components-ember` dependency?

**Comment by bmac** [comment]
I think an optionalDependency would work best.

**Comment by bmac** [comment]
Thanks @CNDW your change is now included in the `2.4.3` and `2.5.0-beta.3` releases https://github.com/components/ember-data/blob/0b4a3e3092604fe3ce54955dfb60dc96d846c228/package.json#L9-L11
